src/website/views/components/data-table/examples/portal/_complex.pug

```diff
@@ -10,15 +10,8 @@ h3 Complex
             :config="config",
             :data="table",
           )
-            template(#statusdesc='payload')
-              i(:class="`chi-icon icon-${payload.due.icon} -icon--${payload.due.color}`" aria-hidden="true")
-              span.-text--truncate(style="padding-left: 0.5rem;") {{ payload.due.status }}
-              br
-              i(:class="`chi-icon icon-${payload.overdue.icon} -icon--${payload.overdue.color}`" aria-hidden="true")
-              span.-text--truncate(style="padding-left: 0.5rem;") {{ payload.overdue.status }}
             template(#status="payload")
-              i(:class="`chi-icon icon-${payload.icon} -icon--${payload.color}`" aria-hidden="true")
```

Comment:
Why do we remove the template here? This makes the column cells render empty. The idea was to simplify the column description only.
Also, can you check what's wrong with the code snippet wrapper? It renders like this for me.
<img width="1792" alt="Screenshot 2022-02-25 at 16 19 09" src="https://user-images.githubusercontent.com/25136131/155740495-97ef97d3-c77a-4294-8de7-fc334b6ba539.png">

Reply:
My bad! I have updated it as required. This should be resolved now.